### ISSUE TITLE
Sign up\Log in should be improvised 

### ISSUE THREAD

**Issue by tanya070820** [khannakshat7/Elektra#144]
## **Is your feature request related to a problem? Please describe.**

- Yes it is related to the problem as their are many such details which are not in sign up/log in form and I also want to add different fonts in it .

## **Describe the solution you'd like**

I would like  to add the district name ,city \town name ,Contact No nd also want to add new font in it for good presentation.
Please try to assign me this issue.

**Comment by khannakshat7** [comment]
@tanya070820 Yeah sure but can u give a better design first so that we can think over it.
You can give a simple idea of how it will look

**Comment by tanya070820** [comment]
@khannakshat7 Yes Sir I will add different fields in it.
